include/deal.II/base/mpi.templates.h

```diff
@@ -264,6 +264,26 @@ namespace Utilities
 
 
 
+    template <typename Number>
+    void
+    sum (const SparseMatrix<Number> &local,
+         const MPI_Comm &mpi_communicator,
+         SparseMatrix<Number> &global)
+    {
+      const SparsityPattern &sp = local.get_sparsity_pattern();
+      Assert(sp == global.get_sparsity_pattern(),
+             ExcMessage("Sparsity pattern of local and global matrices should match."));
```

Comment:
"THE sparsity pattern of THE local and THE global matrix should match."

Reply:
yes, having no articles in mother tongue really makes my life difficult 😄